internal/test/e2e/e2e_test.go

```diff
@@ -240,23 +230,9 @@ func installOperator(t *testing.T, kubectl *kubectlContext, kind *cluster.Contex
 
 	wg.Wait()
 
-	// Deploy the operator.
-	t.Log("Applying operator")
-	err := kubectl.Apply("--kustomize", filepath.Join(*fRepoRoot, "config", "test"))
-	require.NoError(t, err)
-
-	// Ensure the operator starts.
-	err = try.Eventually(func() error {
-		out, err := kubectl.Get("--namespace", "eco-system", "deploy", "eco-controller-manager", "-o=jsonpath='{.status.readyReplicas}'")
-		if err != nil {
-			return err
-		}
-		if out != "'1'" {
-			return errors.New("expected exactly 1 replica of the operator to be available, got: " + out)
-		}
-		return nil
-	}, time.Minute, time.Second*5)
-	require.NoError(t, err)
+	t.Log("Deploying controller")
+	out, err := exec.CommandContext(ctx, "make", "-C", *fRepoRoot, "deploy-controller").CombinedOutput()
+	require.NoError(t, err, string(out))
```

Comment:
Again, I just call back into the Makefile which has the benefit of testing that make target and avoids having two ways of deployging the controller...which might get out of sync.